src/FileSystem-Disk/DiskStore.class.st

```diff
@@ -128,26 +121,26 @@ DiskStore >> basicEntry: directoryEntry path: aPath nodesDo: aBlock [
 		
 	index := 1.
 	pathString := self stringFromPath: aPath.
-	encodedPathString := Primitives encode: pathString.
-	entry := Primitives lookupEntryIn: encodedPathString index: index.
+	encodedPathString := File encode: pathString.
+	entry := File lookupEntryIn: encodedPathString index: index.
 	entry = #badDirectoryPath ifTrue: [ ^ self signalDirectoryDoesNotExist: aPath ].
 	
 	[ entry isNil ]
 		whileFalse: [ 
-			entry at: 1 put: (Primitives decode: entry first).
+			entry at: 1 put: (File decode: entry first).
 			aBlock value: entry.
 			index := index + 1.
-			entry := Primitives lookupEntryIn: encodedPathString index: index ].
+			entry := File lookupEntryIn: encodedPathString index: index ].
 ]
 
 { #category : #public }
 DiskStore >> basicEntryAt: aPath [
 	| encodedPath encodedBasename |
 	
-	encodedPath := Primitives encode: (self stringFromPath: aPath parent).
-	encodedBasename := Primitives encode: aPath basename.
+	encodedPath := File encode: (self stringFromPath: aPath parent).
```

Comment:
I think it would be better to name encoding methods more explicitly like:
- encodePathString:
- decodePathString: